src/models/WeekDateObject.ts

```diff
@@ -22,19 +22,43 @@ export default class WeekDateObject implements AtomicDateObject {
   month: Date;
 
   private dateTypeFormat: DateTypeFormat
+  private today: Date
 
-  static createInnerHTML = (week: AtomicDateObject[], splitWeek = false, month?: number): string => {
+  static datesAreSameDay = (candidateDay: Date, today: Date): boolean => {
+    if ((candidateDay.getFullYear() !== today.getFullYear())
+      || (candidateDay.getMonth() !== today.getMonth())
+      || (candidateDay.getDate() !== today.getDate())) {
+      return false;
+    }
+
+    return true
+  }
```

Comment:
Cleaner, but less readable?
```
static datesAreSameDay = (candidateDay: Date, today: Date): boolean => !(
  candidateDay.getFullYear() !== today.getFullYear()
    || candidateDay.getMonth() !== today.getMonth()
    || candidateDay.getDate() !== today.getDate()
)
```

Reply:
hahaha but worth looking at!
thanks!

Reply:
Yessssss... The whole "if this is true return false" makes me itchy. But so does using a conditional to return true/false in an arrow function in general. And so does my so-easy-to-miss exclamation point... 🤷

Reply:
What about the arrow function makes boolean return values itchy for you?

Reply:
Yeah, that was weirdly specific. :stuck_out_tongue_winking_eye:

Arrow functions using an unnecessary `return` _at all_ seems inelegant at best, but _two_ `return` statements for simple true/false seems very crude.